### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -10973,6 +10974,7 @@ polysaccaride->polysaccharide
 polysaccharid->polysaccharide
 pomegranite->pomegranate
 pomotion->promotion
+pompay->pompeii
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Capitalise the correction.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -11009,19 +11013,22 @@ porgrammeer->programmer
...
 porperty->property
+porsalin->porcelain
 porshan->portion
 porshon->portion
 portait->portrait
 portaits->portraits
 portayed->portrayed
 portguese->Portuguese
 portioon->portion
+portrail->portrayal
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest portrait.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**